mypy/server/update.py

```diff
@@ -170,9 +170,7 @@ def __init__(self,
         # for the cache. This is kind of a hack and it might be better to have
         # this directly reflected in load_graph's interface.
         self.options.cache_dir = os.devnull
-        mark_all_meta_as_memory_only(graph, manager)
-        manager.saved_cache = preserve_full_cache(graph, manager)
-        self.type_maps = extract_type_maps(graph)
+        manager.saved_cache = {}
```

Comment:
Update the docstring of `BuildManager` to reflect the fact that `saved_cache` is no longer used in fine-grained incremental mode.

Reply:
Will do. 

Once this is landed I'm also going to just totally rip out the coarse-grained daemon and all of the `saved_cache` machinery entirely.